Clean Swift/Scene.xctemplate/UIViewController/___FILEBASENAME___Worker.swift

```diff
@@ -26,10 +28,22 @@ class ___VARIABLE_sceneName___Worker {
         }
     }
 
+    // MARK: Perform ___VARIABLE_sceneName___
+
     func perform___VARIABLE_sceneName___(completion: @escaping (Bool, ___VARIABLE_sceneName___Models.Error<ErrorType>?) -> Void) {
         let isSuccessful = true
         let error: ___VARIABLE_sceneName___Models.Error<ErrorType>? = nil
 
         completion(isSuccessful, error)
     }
+
+    // MARK: Track Analytics
+
+    func trackAnalytics(event: ___VARIABLE_sceneName___Models.AnalyticsEvents) {
```

Comment:
@SentulAsia Suggest to reorganise `trackAnalytics` to come before the `perform<SceneName>` method, so that it is consistent with logic protocols order.

```swift
class Worker {
    func validate()
    func trackAnalytics()
    func perform<SceneName>()
}
```

Reply:
will reflect in next PR, sorted the spec and test file as well...